portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns HTML that contains the page view count. */
+@WebServlet("/index1.html")
```

Comment:
not blocking. 
how about "/index.html" or a more meaningful name?

Reply:
Oh okay, I changed to mypage.html.